crates/ruff_linter/src/message/text.rs

```diff
@@ -141,11 +141,13 @@ impl Display for RuleCodeAndBody<'_> {
         let kind = &self.message.kind;
 
         if self.show_fix_status && self.message.fix.is_some() {
+            let indicator = self.message.fix.as_ref().unwrap().applicability().symbol();
```

Comment:
Can we use an `if-let` here instead of `.unwrap()`? We already test `.is_some()` above, we should be able to do `if let Some(fix) = message.fix.as_ref()` instead of `.is_some()` + `.unwrap()`.

Reply:
Yeah I'm actually in the middle of changing this already :)